node/query-neon-postgres/src/neon.js

```diff
@@ -0,0 +1,48 @@
+import postgres from 'postgres';
+
+/**
+ * Throws an error if any of the keys are missing from the object
+ */
+export async function getClient() {
+    const { PGHOST, PGDATABASE, PGUSER, PGPASSWORD, ENDPOINT_ID } = process.env;
+
+    return postgres({
+        host: PGHOST,
+        database: PGDATABASE,
+        username: PGUSER,
+        password: PGPASSWORD,
+        port: 5432,
```

Comment:
dont think this should be hardcoded

Reply:
It came from code example from Neon Console.

Reply:
okay 👍